src/opm/io/eclipse/ESmry_write_rsm.cpp

```diff
@@ -0,0 +1,198 @@
+/*
+   Copyright 2020 Equinor ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#include <opm/io/eclipse/ESmry.hpp>
+
+#include <cmath>
+
+#include <algorithm>
+#include <fstream>
+#include <functional>
+#include <iomanip>
+#include <list>
+#include <ostream>
+#include <regex>
+#include <string>
+
+#include <opm/common/ErrorMacros.hpp>
+
+namespace {
+
+    constexpr std::size_t column_width { 8 } ;
+    constexpr std::size_t column_space { 5 } ;
+
+    constexpr std::size_t column_count { 10 } ;
+
+    constexpr std::size_t total_column { column_width + column_space } ;
+    constexpr std::size_t total_width  { total_column * column_count } ;
+
+    const std::string version_line { } ;
+    // the fact that the dashed header line has 127 rather than 130 dashes has no provenance
+    const std::string divider_line { std::string(total_width - 3, '-') } ;
+
+    const std::string block_header_line(const std::string& run_name, const std::string& comment) {
+        return "SUMMARY OF RUN " + run_name + " OPM FLOW VERSION 1910 " + comment;
```

Comment:
Just remove the "VERSION 1910" part

Reply:
I think we should have the OPM Flow version that created the report in the heading so that we can track changes/errors. I assume this is available via a variable as it is printed out on the terminal report.

Reply:
> I assume this is available via a variable as it is printed out on the terminal report.

Well yeas and no - it is available when compiling opm-simulators, it is not available when compiling opm-common.

Reply:
Removed for now, can be readded as a later enhancement.